### PR TITLE
formicactl: Add stop and destroy command

### PR DESCRIPTION
**Pull request by zeisss**
This PR adds the stop and destroy command. Its actually a copy and paste of the `formicactl start` command.

I guess #28 becomes more interesting with this, as a destroy on a running group may leave artifacts.

### CODE REVIEW

**Comment by zeisss** [comment]
ping @giantswarm/cena

File: controller/controller_test.go [hunk 1]
```diff
@@ -312,6 +312,70 @@ func TestController_Start(t *testing.T) {
...
+		},
+		nil,
+	).Once()
+	fleetMock.On("Destroy", "test-main@1.service").Return(nil).Once()
+	fleetMock.On("Destroy", "test-sidekick@1.service").Return(nil).Once()
+
+	// Execute test
+	req := Request{
+		Group:    "test",
+		SliceIDs: []string{"1"},
```
Comment (xh3b4sd) [inline controller/controller_test.go:338]: It looks still weird without the unit name.

Comment (zeisss) [inline controller/controller_test.go:338]: What unit name?

Comment (xh3b4sd) [inline controller/controller_test.go:338]: Discussed. The actual reason why there is no unit name required and why no `Unit` is given, is the fact that `Destroy` and certain calls do not require the unit name at all. The commands lookup the groups and fetch unit names by the given slice ids to operate on what they found.

Comment (zeisss) [inline controller/controller_test.go:338]: We also talked about that we maybe want a separate Request struct for the Submit case so the `Unit` field is not always there. See https://github.com/giantswarm/giantswarm/issues/527

**Comment by zeisss** [comment]
@giantswarm/cena this PR decreases the coverage because we have no CLI tests. I would not care.

**Comment by xh3b4sd** [comment]
LGTM

**Pull request merged**